examples/complete/main.tf

```diff
@@ -22,13 +22,13 @@ module "subnets" {
 }
 
 module "ec2_instance" {
-  source  = "cloudposse/ec2-instance/aws"
-  version = "0.38.0"
-
+  source          = "cloudposse/ec2-instance/aws"
+  version         = "0.38.0"
+  name            = "myfirstpr"
```

Comment:
This name can be setup with your own fixtures.tfvars file, let's not add this to the PR